docs/block-api.md

```diff
@@ -93,7 +93,7 @@ attributes: {
 },
 ```
 
-* **See: [Attributes](/reference/attributes/).**
```

Comment:
Can't we have something transforming the links automatically during build step when targeting markdown files referenced in the manifest? cc @pento

Reply:
I considered also using relative links, e.g. `../reference/attributes`

Reply:
Relative links are the better option, I think. Less processing is always better. 🙂

Reply:
The advantage of the markdown option is that navigating in Github works :)

Reply:
>The advantage of the markdown option is that navigating in Github works :).

I expect it would have 404'd in GitHub prior to this change as well.